bin/deposit.sh

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env bash
 
+echo "Running bin/deposit.sh"
+
 # Source env settings
 . .env
 . services/ir/.ir.env
 source bin/helper.sh
 
 # NeoGo binary path.
-NEOGO="${NEOGO:-docker exec -it main_chain neo-go}"
+NEOGO="${NEOGO:-docker exec main_chain neo-go}"
 # Wallet file to use for deposit GAS from
 WALLET="${WALLET:-wallets/wallet.json}"
-# Wallet password that would be entered automatically; '-' means no password
-PASSWD="-"
+CONFIG="${CONFIG:-/wallets/empty-config.yml}"
```

Comment:
Why do we have a different name here?
Also is it possible to deposit funds to a wallet with non-empty password, given that config is stored in the container?

Reply:
The config added by mistake, fixed. Now `node-wallet.json` is used for funds deposit.